admin_guide/cluster_autoscaler.adoc

```diff
@@ -0,0 +1,527 @@
+[[admin-guide-cluster-autoscaler]]
+= Managing the Cluster Auto Scaler
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[managing-autoscaler-overview]]
+== Overview
+
+This topic describes setting up the auto-scaler on your
+{product-title} cluster. The auto-scaler's remit is to ensure your
+pods have somewhere to run, and to also ensure there are no unneeded
+nodes when demand drops.
+
+Note: this feature is currently in tech-preview and the this guide
+only applies to running the auto-scaler on AWS.
+
+=== Auto Scale Groups
+
+An Auto Scale Group (ASG) is a logical representation of a set of
+machines. An ASG is configured with a minimum number of instances to
+run, a maximum number of instances that can be run and also the
+desired number of instances to run. An ASG starts by launching enough
+instances to meet its desired capacity. ASGs may start with a minimum
+of zero instances.
+
+=== Launch Configurations
+
+A Launch Configuration (LC) is a template that an ASG uses to launch
+instances; you specify information such as the ID of the Amazon
+Machine Image (AMI), the instance type (e.g., m4.large), a key pair,
+one or more security groups, subnets, etc.
```

Comment:
Is this a standard LC file, or are there limitations or other special requirements?

Reply:
All defined by AWS. So, LCs/ASGs are specific to AWS. Moving to other cloud provides there will be different constraints/parameters that need to be set (when we get there).